detekt-rules-documentation/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/documentation/AbsentOrWrongFileLicense.kt

```diff
@@ -24,7 +24,7 @@ class AbsentOrWrongFileLicense(config: Config = Config.empty) : Rule(config) {
     override val issue = Issue(
         id = RULE_NAME,
         severity = Severity.Maintainability,
-        description = "License text is absent or incorrect in the file.",
+        description = "License text for the given source code file is absent or incorrect.",
```

Comment:
It's your call but I think `in the file` is a bit redundant: The file names are printed out in the log.
```suggestion
        description = "License text for the given source code file is absent or incorrect.",
```

Reply:
```suggestion
        description = "License text is absent or incorrect.",
```